package.json

```diff
@@ -73,5 +73,11 @@
   },
   "workspaces": [
     "./packages/*"
-  ]
-}
\ No newline at end of file
+  ],
+  "volta": {
+    "node": "18.20.8"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.1692.0"
```

Comment:
Dependencies are not added to the root package.json but to the package,json in the module inside `packages/` where it is needed.